.env

```diff
@@ -27,6 +27,7 @@ SEARXNG_QUERY_URL=# where '<query>' will be replaced with query keywords see htt
 
 WEBSEARCH_ALLOWLIST=`[]` # if it's defined, allow websites from only this list.
 WEBSEARCH_BLOCKLIST=`[]` # if it's defined, block websites from this list.
+WEBSEARCH_JAVASCRIPT=false # CPU usage reduces by 60% on average by disabling javascript. Enable to improve website compatibility
```

Comment:
btw, is this variable still used? Couldn't find its usage from a search

Reply:
wait a minute, it was named wrong in the code :joy: fixed